### PR TITLE
Added some small grammar corrections.

### PR DESCRIPTION
**Pull request by AndreeaEne**

### CODE REVIEW

**Review by yarwest — CHANGES_REQUESTED** [review]
There is still bad grammar in a number of places

**Comment by kocsismate** [comment]
Hi Andrea,

I just have accepted another PR for the read me so now, your have some conflicts. :S Could you please resolve them? Sorry for the inconvenience.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 **Woohoo Labs. Harmony is a flexible micro-framework developed for PHP applications.**
 
-Our aim was to create an invisible, easily extensible, but first of all, extremely flexible framework for your
-quality application. We wanted to give you total control via
+Our aim was to create an invisible, easily extensible, ande extremely flexible framework for your
```
Comment (kocsismate) [inline README.md]: typo: ande

File: README.md [hunk 2]
```diff
@@ -43,41 +43,40 @@ This post summarizes the best why Harmony was born: http://www.catonmat.net/blog
...
-So what is the purpose of yet another library with the same functionality?
+[Slim Framework 3](http://www.slimframework.com/docs/concepts/middleware.html), or [Relay](http://relayphp.com/).
+So what's the purpose of yet another library with the same functionality?
 
 We believe Harmony is superior to the others in two key things:
 
-- It is the simplest of all. Although simplicity is subjective, one thing is for sure: Harmony offers the least
-functionality which is minimally needed. It doesn't have capabilities which are not truly required. That's why Harmony
+- It is the simplest of all. Although simplicity is subjective, we know one thing for sure: Harmony offers the least
+functionality that's is minimally needed. It doesn't have any capabilities which are not truly required. That's why Harmony
```
Comment (kocsismate) [inline README.md]: another small typo: "that's is"

**Review by kocsismate — CHANGES_REQUESTED** [review]
Thanks for your time! I like your changes very much :)

**Comment by AndreeaEne** [comment]
Done! :)

**Comment by kocsismate** [comment]
Thank you! I've just seen that you resolved the conflicts but I still can't accept your PR automatically. :S Do you have any idea why this happens?

**Comment by AndreeaEne** [comment]
Probably because I changed some of the conflicted words as well.

**Pull request closed without merging**